cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,23 @@
-describe('', () => {
-  before(() => {
-
-  });
+describe('Conduit article', () => {
+  let user;
+  let article;
 
-  it('', () => {
+  before(() => {
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+    });
 
+    cy.task('generateArticle').then((generatedArticle) => {
+      article = generatedArticle;
+    });
   });
 
-  it('', () => {
-
+  it('should create an article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.createArticle(article.title, article.description, article.body);
+    cy.visit('profile/' + (user.username).toLowerCase());
+    cy.get('h1').click();
+    cy.get('button').contains('Delete Article').click();
+    cy.get('.article-preview').should('contain', 'No articles are here... yet.');
   });
```

Comment:
There should be two tests in total:
1. Login with a function, create an article manually, assert that the article has been created.
2. Loin with a function, create an article with a function, delete the article manually, assert that the article has been deleted.